lib/wanda_web/router.ex

```diff
@@ -57,14 +57,18 @@ defmodule WandaWeb.Router do
         get "/groups/:id/executions/last", ExecutionController, :last
         post "/executions/start", ExecutionController, :start
         get "/catalog", CatalogController, :catalog
+      end
+
+      scope "/groups" do
+        pipe_through [:api_v1, :protected_api]
 
-        get "/groups/:group_id/catalog", CatalogController, :selectable_checks
+        get "/:group_id/checks", CatalogController, :selectable_checks
 
-        post "/:check_id/customize/:group_id",
+        post "/:group_id/check/:check_id/customization",
```

Comment:
question: shouldn't this and the delete have `checks` instead of `check`?

Reply:
whoopsie